StaticAnalyzer/views/android.py

```diff
@@ -660,6 +679,9 @@ def StaticAnalyzer(request):
                                                                   ANDROVER=ANDROVER,
                                                                   MANIFEST_ANAL=MANIFEST_ANAL,
                                                                   PERMISSIONS=PERMISSIONS,
+# Esteve 21.08.2016 - begin - Permission Analysis with Androguard
+                                                                  ANDROPERMS = "",
```

Comment:
Hi Ajin. The reason for this would be the same as the reason why STRINGS (strings from apk) is also assigned to an empty string some few code lines below. Strings_from_apk can only be used when the input to MobSF is an APK, and not a ZIP. The same issue would apply to the androlyze tool of the Androguard framework.